deploy/testnet/revert.sh

```diff
@@ -16,7 +16,7 @@ source ./config.env
 
 container_name="honest_worker"
 chainspec="./chainspec.json"
-rpc_api_endpoint="http://127.0.0.1:9944"
+export rpc_api_endpoint="http://127.0.0.1:9944"
```

Comment:
Why we need `export` here?

Reply:
In deploy/testnet/revert.sh line 19:
rpc_api_endpoint="http://127.0.0.1:9944"
^--------------^ SC2034 (warning): rpc_api_endpoint appears unused. Verify use (or export if used externally).

Reply:
Variable is unused in this script. If this is true, then it can be removed entirely.